packages/notifi-react-card/lib/components/intercom/IntercomCard.tsx

```diff
@@ -91,6 +92,9 @@ export const IntercomCard: React.FC<
           if (participant.conversationParticipantType === 'MEMBER') {
             setUserId(participant.profile.id);
           }
+          if (participant.conversationParticipantType === 'SUPPORT') {
+            setSupportAvatarUrl(participant.profile.avatarData);
```

Comment:
You may want to double check that the avatardatatype is url here before setting it.